shell/platform/android/platform_view_android_jni.cc

```diff
@@ -484,6 +485,26 @@ static void InvokePlatformMessageEmptyResponseCallback(JNIEnv* env,
       );
 }
 
+static jint GetTextOffsetBefore(JNIEnv* env,
```

Comment:
Can you add a tiny docstring for this function?

Reply:
done